gpflow/utilities/utilities.py

```diff
@@ -230,6 +231,33 @@ def update_state(parameter_or_variable, path, state):
     return state
 
 
+if Version(tfp.__version__) > Version("0.11.0"):
```

Comment:
Not sure, but do you mean this?
```suggestion
if Version(tfp.__version__) < Version("0.11.0"):
```